README.md

```diff
@@ -65,55 +65,65 @@ with:
         redeemTask: true
       dataPurge:
       formattingEvaluator:
-        scores:
-          br: 0
-          code: 1
-          p: 1
-          em: 0
-          img: 0
-          strong: 0
-          blockquote: 0
-          h1: 1
-          h2: 1
-          h3: 1
-          h4: 1
-          h5: 1
-          h6: 1
-          a: 1
-          li: 1
-          td: 1
-          hr: 0
         multipliers:
           - select: [ ISSUE_SPECIFICATION ]
             formattingMultiplier: 1
-            wordValue: 0.1
+            symbols:
+              "\\b\\w+\\b": 0.1
+            scores: # Scores can be set for each item differently
```

Comment:
maybe it'd be better to name this `tags` or `tagMultiplier`?

Reply:
Named it after the spec, I can change it. @0x4007 What do you prefer?

Reply:
Not sure, perhaps `pattern: ` however I know I had another task somewhere for custom aliases to map to specific regexes so that its more readable in the config. Or we can consider hard coding these inside of the plugin. For example:

```yml
alias: 
   "\\b\\w+\\b": "words"
```

And then later in the config we can just do something like
```yml
pattern: 
   words: 0.1
```

I have a feeling hard coding the common useful ones might be best

Reply:
I can do that but I have to change the configuration schema again since the spec had the `scores` (or `bonus` here) per selector not per the regex in the selector.

Reply:
My goal is to make the config as intuitive as possible, sorry for the runaround.

Reply:
Oken then I'll change the whole config shape, will put that PR in a draft again because it will take a while.